Project.toml

```
@@ -11,15 +11,16 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 StatsFuns = "4c63d2b9-4356-54db-8cca-17b64c39e42c"
 
 [compat]
-Distributions = "0.14 - 0.24"
-PDMats = "0.6 - 0.10"
-StatsFuns = "0.6 - 0.9"
-julia = "0.7, 1"
+Distributions = "0.25"
+PDMats = "0.11"
+SpecialFunctions = "2.1"
+StatsFuns = "0.9"
```

Meanwhile StatsFuns 1 was released.
```suggestion
StatsFuns = "0.9, 1"
```
Alternatively, one could use the same bounds as Distributions:
```suggestion
StatsFuns = "0.9.15, 1"
```